README.md

```diff
@@ -199,7 +200,7 @@ Note that you might see an `osiris.deislabs.io/selector` annotation - this is fo
 Deploy the [example application](example/hello-osiris.yaml) `hello-osiris` :
 
 ```
-kubectl create -f ./example/hello-osiris.yaml
+kubectl create -f ./example/hello-osiris-deployment.yaml
```

Comment:
If we're going back to just having one example, we should probably go back to not including "deployment" in this file name.

Reply:
good catch! I just fixed it